src/aleph/vm/orchestrator/views/__init__.py

```diff
@@ -319,3 +319,61 @@ async def update_allocations(request: web.Request):
         },
         status=status_code,
     )
+
+
+async def notify_allocation(request: web.Request):
+    """Notify instance allocation, only used for Pay as you Go feature"""
+    if not authenticate_api_request(request):
+        return web.HTTPUnauthorized(text="Authentication token received is invalid")
```

Comment:
Why do we ask for scheduler authentication here ?

Reply:
This function should however check the balance of the flow.

Reply:
https://github.com/aleph-im/aleph-vm/pull/517/commits/56f09ec95a9a9bc2c10842ae77628c619cc20ab2